firebase-sessions/src/main/kotlin/com/google/firebase/sessions/SessionsSettings.kt

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright 2023 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.firebase.sessions
+
+import kotlin.time.Duration
+import kotlin.time.Duration.Companion.minutes
+
+/**
+ * [SessionsSettings] manages all the configs that are relevant to the sessions library.
+ *
+ * @hide
+ */
+internal class SessionsSettings {
+  // Setting to qualify if sessions service is enabled.
+  internal val sessionsEnabled: Boolean
```

Comment:
I like this setup of implementing them in the getters. I don't think we need to say `internal` on each one through because the class is internal. It'll look cleaner IMO.

Reply:
Done.